classes/security/UserGroupDAO.inc.php

```diff
@@ -910,7 +910,7 @@ function _getSearchSql($searchType, $search, $searchMatch, &$params) {
 			}
 		}
 
-		$searchSql .= $this->userDao->getOrderBy(); // FIXME Add "sort field" parameter?
+		$searchSql .= ' ORDER BY u.user_id'; // FIXME Add "sort field" parameter?
```

Comment:
I was going to update the `getOrderBy()` method, but I didn't check all of its usages, so I just updated here.
This piece improves the performance from 8s to 2s in my use-case.

Reply:
This will mean that several features will lose their current sorting (by the user's name):
- Subscriber selection in the subscription/payments area (`controllers/grid/users/subscriberSelect/SubscriberSelectGridHandler.inc.php`)
- User grid in settings (`lib/pkp/controllers/grid/settings/user/UserGridHandler.inc.php`)
- User list in user export plugin (`lib/pkp/controllers/grid/users/exportableUsers/ExportableUsersGridHandler.inc.php`)
I suspect this will make those lists less usable. Maybe this could be applied as a stop-gap for better optimization to installations that are encountering performance problems? I'm hesitant to merge it.

Reply:
Ok! I just wanted you to balance the pros/cons of doing it, as it's an easy improvement, on interfaces with a search by name, it's less of an issue, but I also prefer to keep this natural sorting 😅
I have a bunch of ideas, I'll check if I can come up with something simple to improve this.